src/_C002.ino

```diff
@@ -56,9 +56,9 @@ bool CPlugin_002(byte function, struct EventStruct *event, String& string)
         // Find first enabled controller index with this protocol
         byte ControllerID = findFirstEnabledControllerWithId(CPLUGIN_ID_002);
         if (ControllerID < CONTROLLER_MAX) {
-          StaticJsonBuffer<512> jsonBuffer;
-          JsonObject& root = jsonBuffer.parseObject(event->String2.c_str());
-          if (root.success())
+          DynamicJsonDocument root(1024);
```

Comment:
This should be 512

Reply:
Do you want to change it before I merge it?

Reply:
C009 is tested but C002 was not, so I was waiting for someone to comment on this. The original code had this size set to 512, I dont know why I set it to 1024.